Packs/FetchIndicatorsFromFile/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py

```diff
@@ -128,18 +128,9 @@ def detect_type(indicator):
     if re.match(ipv6Regex, indicator):
```

Comment:
add the if of the 3 as i said in the previous PR

Reply:
done